src/share/classes/com/sun/javatest/regtest/agent/JUnitRunner.java

```diff
@@ -232,7 +232,7 @@ public void executionStarted(TestIdentifier identifier) {
 
         @Override
         public void executionFinished(TestIdentifier identifier, TestExecutionResult result) {
-            if (identifier.isTest()) {
+            /* if (identifier.isTest()) */ {
```

Comment:
Remove commented out code, and maybe the remaining curly braces?

Reply:
My idea was to keep indentation changes minimal.

Reply:
Removed unused/commented out code and fixed indentation via https://github.com/openjdk/jtreg/pull/147/commits/cb8bbb07a9de3d07d3ed759b32d230425183c2c1

Reply:
In this case, neater code is less confusing. Thanks for the update